ground/src/test/java/com/google/android/ground/persistence/remote/firebase/schema/SubmissionMutationConverterTest.kt

```diff
@@ -234,7 +235,10 @@ class SubmissionMutationConverterTest {
       ),
       mapOf(
         MULTIPLE_CHOICE_RESPONSES_FIELD_NUMBER.toString() to
-          mapOf(OTHER_TEXT_FIELD_NUMBER.toString() to "other value"),
+          mapOf(
+            OTHER_TEXT_FIELD_NUMBER.toString() to "other value",
+            OTHER_SELECTED_FIELD_NUMBER.toString() to true,
+          ),
         TASK_ID_FIELD_NUMBER.toString() to "single_choice_task_other",
```

Comment:
Can we add one more test case where the other text is empty?

Reply:
Done